.gitignore

```diff
@@ -56,3 +56,7 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Virtual environments
+venv/
```

Comment:
When working on a project, I generally do `virtualenv venv` in the main directory and develop using that environment. I know others who use `.venv`. I'm under the impression this is somewhat standard practice?